basic-sql-commands.md

```diff
@@ -40,8 +40,8 @@ SHOW DATABASES;
 +----------+
 | Database |
 +----------+
-| db1      |
-| system   |
+| "db1"    |
```

Comment:
This occurred with the switch to the pgwire driver. We can probably get rid of them as the formatting is up to us (currently we blindly use `%q` for any `[]byte` returned).